Jet_fitting_3/include/CGAL/Monge_via_jet_fitting.h

```diff
@@ -758,14 +803,9 @@ void Monge_via_jet_fitting<DataKernel, LocalKernel, SvdTraits>::
 switch_to_direct_orientation(Vector_3& v1, const Vector_3& v2,
 			    const Vector_3& v3) 
 {
-  typedef typename CGAL::Linear_algebraCd<FT>::Matrix Matrix;
-  Matrix M(3,3);
-  for (int i=0; i<3; i++) M(i,0) = v1[i];
-  for (int i=0; i<3; i++) M(i,1) = v2[i];
-  for (int i=0; i<3; i++) M(i,2) = v3[i];
-
-  CGAL::Sign orientation = CGAL::Linear_algebraCd<FT>::sign_of_determinant(M);
-  if (orientation == CGAL::NEGATIVE) v1 = -v1;
+  Vector_3 v = cross_product(v2,v3);
+  double d = v*v1;
+  if(d < 0)v1 = -v1;
 }
 
```

Comment:
Here we have a function that was using a predicate that is now using a construction. If the underlying kernel does not have exact construction this might lead to errors.